llvm/test/Transforms/DeadStoreElimination/offsetted-overlapping-stores.ll

```diff
@@ -47,6 +47,29 @@ bb:
   ret void
 }
 
+define void @ScalableVectorTestFullyOverlapping(ptr %arg, i32 %i) {
```

Comment:
It would probably be good to add negative tests here as well and combinations where the first store is scalable and different constant GEP offsets

Reply:
Done. Note that there is no test-update here, due to a deficiency in AA, which I expect will be fixed by #110519.